docs/user_guide.md

```diff
@@ -248,3 +249,75 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
 
 When an image is signed, it has a tick shown in UI; otherwise, a cross sign(X) is displayed instead.  
 ![browse project](img/content_trust.png)
+
+### Vulnerability scan via Clair 
+**CAUTION: Clair is an optional installing component, please make sure you have already enable it in your Harbor before you go through this part!**

```

Comment:
NOTE: Clair is an optional component, please make sure you have already installed it in your Harbor instance before you go through this section.